src/software/ai/motion_constraint/motion_constraint_manager_test.cpp

```diff
@@ -0,0 +1,226 @@
+/**
+ * This file contains the unit tests for functions in motion_constraint_manager.cpp
+ */
+
+#include "software/ai/motion_constraint/motion_constraint_manager.h"
+
+#include <gtest/gtest.h>
+
+#include <set>
+
+#include "software/ai/hl/stp/tactic/all_tactics.h"
+#include "software/ai/world/game_state.h"
+
+namespace
+{
+    // vector of pairs of Tactic and whitelists
+    std::vector<std::pair<Tactic*, std::set<MotionConstraint>>> test_vector = {
+        std::pair<Tactic*, std::set<MotionConstraint>>(new MoveTactic(),
+                                                       std::set<MotionConstraint>({}))};
```

Comment:
Eventually, this vector will contain every tactic and its corresponding whitelist and we'll be able to test them all here